README.hebrew.md

```diff
@@ -922,13 +922,12 @@ API, הודעה חדשה נרשמת לתור, וקריאה לכלי צפיה ב
 
 <br/><br/>
 
-## ![✔] 5.11. Get your frontend assets out of Node
+## ![✔] 5.11. הוציאו את קבצי צד הלקוח שלכם מחוץ ל-Node
 
-**אמ;לק:** Serve frontend content using a specialized infrastructure (nginx, S3, CDN) because Node performance gets hurt when dealing with many static files due to its single-threaded model. One exception to this guideline is when doing server-side rendering
+**אמ;לק:** שרתו את תוכן צד הלקוח באמצעות תשתית ייעודית (nginx, S3, CDN) מאחר שהביצועים של Node נפגעים מהתעסקות עם המון קבצים סטטים עקב מודל החוט היחיד. 
+**אחרת:** חוט ה-Node היחיד שלכם יהיה עסוק בהזרמת מאות קבצי html/תמונות/אנגולר/ריאקט במקום להקצות את כל המשאבים שלו עבור המשימה לשמה הוא נועד - לשרת תוכן דינמי.

```

Comment:
Thanks @ShaiDemri to take a part in the Hebrew translation 💪 

1. pls translate also the title in the `table of content` (`תוכן העניינים`)
2. When I looked at the rendering results it seems the a space between the TLDR and otherwise sections are missing. pls fix it too.

great translation